### ISSUE TITLE
[postgres] include table count as a metric

### ISSUE THREAD

**Issue by stefanfoulis** [DataDog/dd-agent#1192]
Amount of tables per database.

Possibly the instructions on how to prepare postgres for datadog would need to be changed, since currently only access to the ``pg_stat_database`` view is granted and the information is not in there.

**Comment by LeoCavaille** [comment]
@stefanfoulis I guess you would like to see info about user tables and not all tables (including toast tables), would something like converting this request to metrics make sense to you ?
```
SELECT count(*), schemaname FROM pg_stat_user_tables GROUP BY schemaname;
```

**Comment by remh** [comment]
Adding this to triage for 5.3.0 release.

**Comment by stefanfoulis** [comment]
@LeoCavaille yep, that looks about right :-)

**Comment by remh** [comment]
Will be done by https://github.com/DataDog/dd-agent/pull/1373

```
postgresql.total_tables
```

**Comment by remh** [comment]
PR is merged, this will go out with 5.3.0.
